query-monitor/classes/Util.php

```diff
@@ -118,9 +118,9 @@ public static function get_file_dirs() {
 			self::$file_dirs = apply_filters( 'qm/component_dirs', self::$file_dirs );
 
 			self::$file_dirs['plugin'] = WP_PLUGIN_DIR;
-			self::$file_dirs['mu-vendor'] = WPMU_PLUGIN_DIR . '/vendor';
-			self::$file_dirs['go-plugin'] = WPMU_PLUGIN_DIR . '/shared-plugins';
-			self::$file_dirs['mu-plugin'] = WPMU_PLUGIN_DIR;
+			self::$file_dirs['mu-vendor'] = WPVIP_MU_PLUGIN_DIR . '/vendor';
+			self::$file_dirs['go-plugin'] = WPVIP_MU_PLUGIN_DIR . '/shared-plugins';
+			self::$file_dirs['mu-plugin'] = WPVIP_MU_PLUGIN_DIR;
```

Comment:
This might be misleading, since we intend to support customer mu plugins at `WPMU_PLUGIN_DIR`.  So we should call this one `vip-mu-plugin` I think and leave `self::$file_dirs['mu-plugin'] = WPMU_PLUGIN_DIR;`.

Reply:
Nice catch!

I fixed it on https://github.com/Automattic/vip-go-mu-plugins/pull/5272/commits/a4cc270ac25cb7580713cd0c9fa686010a97e168 and just added the `self::$file_dirs['mu-plugin'] = WPMU_PLUGIN_DIR;` when `WPVIP_MU_PLUGIN_DIR !== WPVIP_MU_PLUGIN_DIR` to make sure we are not adding the directory twice when the are the same.